### DISCUSSION TITLE
Moving from GitHub to AppStore release on macOS [Question]

### DISCUSSION THREAD

**Discussion by gioboske** [utmapp/UTM#4945] [UTM Help (macOS)] [answered]
Actually a question rather than a real issue.
I've installed UTM v4.1.5 from GitHub on a MacBook Pro 14 (M1 Pro, macOS Ventura) and tested it with a Windows 11 guest system. Now I would move to the AppStore version, so to found the project (and to be able to get the automatic updates).

Which is the recommended way to move from the GitHub to the AppStore version, without loosing the current configured VMs?
Can I simply remove the current .app from the Application folder and install the new version from the AppStore?
Will all the currently configured VMs be available in the AppStore version (all my VMs are currently stored in the default location `~/Library/Containers/com.utmapp.UTM/Data/Documents`)?

Thank you!

**Configuration**  
* UTM Version: 4.1.5
* macOS Version: Ventura (13.1)
* Mac Chip (Intel, M1, ...): M1 Pro

**Comment by osy** [comment, marked as answer]
You don’t have to do anything, they share the same identifier so if you install the app store version, it will automatically overwrite the .app and all the settings/data will be the same.

**Reply by gioboske** [reply]
Thank you very much!
